scripts/toaster_configuration

```diff
@@ -189,8 +189,26 @@ EOF
 }
 
 create_toaster_venv () {
-    python -m virtualenv -p python3 "$BUILDDIR/venv"
-    python -m virtualenv -p python2 "$BUILDDIR/venv"
+    if [ $(lsb_release -i | awk {'print $3'}) == "RedHatEnterpriseWorkstation" ]; then
+        # If its RHEL 7 there is a bug in virtaulenv which has to be tweaked
+        if [ $(lsb_release -r | awk {'print $2'} | cut -d "." -f1) -eq 7 ]; then
+            virtualenv "$BUILDDIR/venv"
+            (
+                . "$BUILDDIR/venv/bin/activate"
+                pip install --upgrade virtualenv
+                python -m virtualenv -p python3 "$BUILDDIR/venv"
+                cd "$BUILDDIR/venv/bin"
+                unlink python
+                unlink python2
+                unlink python2.7
+                cd -
+                python -m virtualenv -p python2 "$BUILDDIR/venv"
+            )
+        fi
+    else
```

Comment:
I'm assuming we need to unlink in order to let virtualenv fully update the environment after upgrading virtualenv?  Could we use --clear, instead? Alternatively, we could install the latest virtualenv into the user local python area and then use that when we construct it originally, rather than installing the new virtualenv into the virtualenv.